source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -236,6 +236,9 @@ def output(self, key, obj):
     "links": TicketLinks(attribute="section_id")
 }
 
+context = {
+    "car_co2_emission": Co2Emission()
```

Comment:
do you really need this ?
doesn't it work with only a flask dict ?
like (speudo code):

```
"car_co2_emission": NonNullNester({
'value': fields.String,
)}
...
```